src/components/EditPhoto.jsx

```diff
@@ -28,10 +28,14 @@ export const EditPhoto = ({ handleUpdate, showInput, shouldShowInput }) => {
       <Button
         variant="contained"
         color="success"
-        style={{margin:"10px"}}
+        style={{ margin: "10px" }}
         onClick={() => {
-          handleUpdate(imageUrl);
-          setImageUrl("");
+          if (imageUrl == undefined) {
```

Comment:
we can try to do a smarter validation, verifying if what the user is inputting is properly a link

Reply:
```suggestion
          if (imageUrl.match(/^(?:(?:https?|ftp):\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,}))\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/i)) {
```